services/web/server/src/simcore_service_webserver/application_settings.py

```diff
@@ -72,9 +73,26 @@ class ApplicationSettings(BaseCustomSettings, MixinLoggingSettings):
 
     # RUNTIME  -----------------------------------------------------------
     # settings defined from environs defined when container runs
-    # NOTE: keep alphabetically if possible
+    #
+    # NOTE: Please keep fields alphabetically if possible
     AIODEBUG_SLOW_DURATION_SECS: float = 0
 
+    # Release information: Passed by the osparc-ops-autodeployer
```

Comment:
all three of this fields are passed form the auto deployer, right?

Reply:
yes